xbmc/pvr/recordings/PVRRecordingsPath.h

```diff
@@ -56,10 +60,12 @@ namespace PVR
 
   private:
     static std::string TrimSlashes(const std::string &strString);
+    size_t GetDirectoryPathStart() const;
 
```

Comment:
Generally, I always document public and protected methods, but not private methods. And with this approach I'm far ahead if many other devs in the project. ;-) I will see whether I can find a more speakingb method name though.